src/lib.rs

```diff
@@ -344,7 +349,7 @@ fn create_cubic_outline<T>(goutline: &GlifOutline) -> Outline<T> {
                 PointType::OffCurve => {
                     stack.push_back(&gp);
                 }
-                PointType::Line | PointType::Curve => {
+                PointType::Line | PointType::Move | PointType::Curve => {
```

Comment:
For posterity here is my oops that caused so much turmoil :-) (mfeq/Qstroke#3 etc.)